### ISSUE TITLE
mat-menu trigger is setting aria-expanded to null instead of false

### ISSUE THREAD

**Issue by apurva-jakhanwal** [angular/components#16859]
#### Reproduction

Steps to reproduce:
1. Add a button with [matTriggerFor] for a mat menu 
```
<button mat-icon-button
        #menuButton
        [matMenuTriggerFor]="exampleMenu">
</button>

<mat-menu #exampleMenu="matMenu">
```
2. Inspect the element in the browser
3. Click on the button to open the menu- it sets the aria-expanded to true
4. Inspect the element when the menu is closed - having it closed does not set it to false (sets it to null)
- https://github.com/angular/components/blob/master/src/material/menu/menu-trigger.ts
- `'[attr.aria-expanded]': 'menuOpen || null',`

 
#### Expected Behavior

When the menu is closed, aria-expanded should be set to false

#### Actual Behavior

When the menu is closed, aria-expanded is not set to false, it is set to null

#### Environment

- Angular: 7.1.3
- CDK/Material: 7.3.0
- Browser(s): Chrome, Edge Chromium, Edge
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by jelbourn** [comment]
This is intentional. Setting to null omits the attribute entirely, which intends to prevent screen-readers from reading necessary information. Are you seeing some issue here with a particular screen reader?

**Comment by adalbertomaia** [comment]
> This is intentional. Setting to null omits the attribute entirely, which intends to prevent screen-readers from reading necessary information. Are you seeing some issue here with a particular screen reader?

It's important to have this information, otherwise how the user will know that's a collapsed field?

**Comment by jelbourn** [comment]
> It's important to have this information, otherwise how the user will know that's a collapsed field?

Via the `aria-haspopup` attribute.

Closing this since AFAICT it's working as intended.
